mods/ts/rules/nod-structures.yaml

```diff
@@ -25,6 +25,7 @@ NAPOWR:
 		Range: 4c0
 		MaxHeightDelta: 3
 	WithIdleOverlay@LIGHTS:
+		RequiresCondition: !disabled
```

Comment:
I was thinking more about EMP, which already grants `disabled`.  But ok, this can be done in a future PR.